index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portofolio</title>
+</head>
+<body>
+    <header class="header">
+      <div class="logo">My Logo</div>
+     
+      <div class="menu" id="menu">
+        <a class="menu-icon" href="#menu">
+            <img src="./assets/images/Enabled.svg" alt="" srcset="">
+          </a>
+        <div class="menu-close">
+            <a href="#" class="menu-close-icon"><img src="./assets/images/times.svg" alt="" srcset=""></a>
+        </div>
+        <div class="menu-items">
+            <a href="#portofolio">Portofolio</a>
+            <a href="#about">About</a>
+            <a href="#contact">Contact</a>
+          </div>
+      </div>
+    </header>
+    <main>
+        <section class="bio">
+            <p class="bio-header">
+                I'm Haben. <br> Glad to see you.
+            </p>
+            <p class="bio-info">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <div class="link">
+                <p class="link-text">LET'S CONNECT</p>
+                <div class="profile-links">
+                    <a href=""><img src="./assets/images/twitter.svg" alt="" srcset=""></a>
+                    <a href=""><img src="./assets/images/m.svg" alt="" srcset=""></a>
+                    <a href="https://github.com/habitoz"><img src="./assets/images/github.svg" target="_blank"  alt="" srcset=""></a>
+                    <a href=""><img src="./assets/images/ok.svg" alt="" srcset=""></a>
+                </div>
+            </div>
```

Comment:
- [ ] Instaed of using a `div` that has `a` tag , kindly use `ul`  and `li` to represent this list of social media icons

```bash
<ul>
      <li></li>
       ...
</ul>
```